### PR TITLE
Update slack link URL on homepage #60

### LINKED ISSUE #60: Update slack link URL on homepage
update the slack link URL button to "go.tampa.dev/slack" 

It should be in 3 places

- one on the header
- the main call to action button on page
- another URL on the icons at the bottom of the page

also double check the links work as expected too

### PR DESCRIPTION
**Pull request by Priyanshii677**

### CODE REVIEW

File: src/_includes/blocks/footer.njk [hunk 1]
```diff
@@ -42,7 +42,7 @@
       <li><a href="https://www.youtube.com/@tampadevs" target="_blank" rel="noopener"><img class="social-icon" src="/_assets/misc/icons/youtube.png" alt="YouTube"/></a></li>
       <li><a href="https://twitter.com/tampadevs" target="_blank" rel="noopener"><img class="social-icon" src="/_assets/misc/icons/twitter.svg" alt="Twitter"/></a></li>
       <li><a href="https://www.meetup.com/tampadevs" target="_blank" rel="noopener"><img class="social-icon" src="/_assets/misc/icons/meetup.svg" alt="Meetup"/></a></li>
-      <li><a href="https://bit.ly/tampadevs-join-slack" target="_blank" rel="noopener"><img class="social-icon" src="/_assets/misc/icons/slack.svg" alt="Slack"/></a></li>
+      <li><a href="go.tampa.dev/slack" target="_blank" rel="noopener"><img class="social-icon" src="/_assets/misc/icons/slack.svg" alt="Slack"/></a></li>
```
Comment (manuelosorio) [inline src/_includes/blocks/footer.njk]: When this is clicked user will be sent to http://tampadevs.com/go.tampa.dev/slack
to fix that issue please change it to https://go.tampa.dev/slack

File: src/index.njk [hunk 2]
```diff
@@ -43,7 +43,7 @@ nav: true
       <a
         id="cta-button-join-slack"
         class="promo__content__button"
-        href="https://join.slack.com/t/tampa-devs/shared_invite/zt-veftezkg-kq~jFtC1FCz4o6suybMl5Q"
+        href="go.tampa.dev/slack"
```
Comment (manuelosorio) [inline src/index.njk]: This one will need that change too.

**Review by manuelosorio — CHANGES_REQUESTED** [review]
Thank you for the pull request. The links just need a slight update to work properly.

**Comment by manuelosorio** [comment]
The navigation slack icon will also need the new url for slack.

That can be updated in this file. https://github.com/TampaDevs/tampadevs/blob/master/src/_data/basics.json

**Comment by chtzvt** [comment]
Great work @Priyanshii677! With the minor tweaks suggested by Manuel we'll be able to merge this immediately.

**Comment by Priyanshii677** [comment]
Here are the new changes @manuelosorio

**Review by manuelosorio — APPROVED** [review]
Thank you for the changes.

**Pull request merged**